src-graphql/collection-query.js

```diff
@@ -0,0 +1,36 @@
+import imageQuery from './image-query';
+import createGid from './create-gid';
+
+export default function collectionQuery({client, id}, fields = ['id', 'handle', 'updatedAt', 'title', ['image', imageQuery()]]) {
```

Comment:
maybe stick the default list in a value so it fits more cleanly on a shorter line? So like 

```javascript
const defaultFields = ['id', 'handle', 'updatedAt', 'title', ['image', imageQuery()]];

export default function collectionQuery({client, id}, fields = defaultFields) {
```